stable/polaris/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: Validation of best practices in your Kubernetes clusters
+name: polaris
+version: 0.1.4
```

Comment:
Normally we version the chart independently of the app.  Seems fine here to start, but going forward I would expect them to diverge.

Reply:
We'd been using `version` as `appVersion` before - agreed they should start to diverge at this point.